### ISSUE TITLE
change_representer for Stiefel (Default/Euclidean metric) and Grassmann (or IsometricEmbeddings in general)

### ISSUE THREAD

**Issue by kellertuer** [JuliaManifolds/Manifolds.jl#572]
I just noticed that we could provide `change_representer` “for free” on Isometric Embeddings, like Stiefel (with its Default Metric / Euclidean Metric) or Grassmann (same metric, i.e. also isometric embedding), since there the change is the identity.

Currently
```
using Manifolds
M = Grassmann(5,3)
p = rand(M)
X = rand(M; vector_at=p)
change_representer(M, E p, X)
```
errors (when falling back to MetricManifold somewhere).

And I think this could even be defined for the `IsIsometricEmbedding` case as being the identity.

**Comment by mateuszbaran** [comment]
Sure, that makes sense. Defining it based on `IsIsometricEmbedding` seems appropriate.

**Comment by kellertuer** [comment]
Since we have the EmbeddedManifold and the Metric in ManifoldsBase, it might even be worth checking whether it can be done there – since it does not even require a MetricManifold

**Comment by mateuszbaran** [comment]
Yes, sure, I think `change_representer` can be defined in ManifoldsBase.jl.

**Comment by kellertuer** [comment]
We do not have the isometric embedding there, but it should be doable to define that case here, now. Will check.

**Comment by kellertuer** [comment]
This as default is not as easy is I thought since we have to make sure the metric you come from is the one from the embedding. The special case for Stiefel/Grassman can be easily done, though.

**Comment by kellertuer** [comment]
This was resolved in #587
